src/script/repositories/conversation/ConversationRepository.ts

```diff
@@ -2456,6 +2457,27 @@ export class ConversationRepository {
     return this.eventRepository.injectEvent(event, EventRepository.SOURCE.INJECTED);
   }
 
+  /**
+   * Will inject a member deleted event in 1:1 conversations with that user.
+   * This will be used to notify the other user that the user was deleted.
+   *
+   * @param userId User ID of the user that was deleted
+   */
+
+  private readonly onUserDeleted = async (userId: QualifiedId) => {
+    const found1to1Conversation = this.conversationState.get1to1ConversationWithUser(userId);
+    if (!found1to1Conversation) {
+      return;
+    }
+
+    const deletedEvent = EventBuilder.buildMemberDeleted(
+      found1to1Conversation,
+      userId,
+      this.serverTimeHandler.toServerTimestamp(),
+    );
+    await this.eventRepository.injectEvent(deletedEvent, EventRepository.SOURCE.INJECTED);
+  };
```

Comment:
inject a `CONVERSATION_EVENT.MEMBER_LEAVE` event in 1:1 conversation with a deleted user